module-3/backend/week_one.md

```diff
@@ -5,22 +5,50 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What is `json`, what does it stand for, and why is it important?
+<p>json is a data type that can be easily interpreted by many languages. It serves as a bridge between ruby and javascript.</p>
 2. What's the difference between `joins` and `includes` in ActiveRecord?
+<p>Joins creates a join table with two tables that already have some kind of relationship. Includes is similar to joins, but rather than creating a new table, it just provides you access to attributes on another table in the background.</p>
```

Comment:
Take another look at includes. Major point here is that includes preloads related data to avoid future queries where possible.